src/stories/input.stories.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import 'styled-components/macro';
+import Input from '../Me/components/Input';
+import Textarea from '../Me/components/Textarea';
+import FormField from '../Me/components/FormField';
+import '../index.css';
```

Comment:
Is this needed (in all files)?

Reply:
that's a good point.
I just copy pasted another story and used it as a template to be honest, so I did not think about it.
We actually need it to have the stories having the same styling of the app, but importing it in  the stories is not a good practice.
I would import it in `.storybook/preview.js` it's imported there once for all the stories.

Reply:
Please do